notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,330 @@
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  direction strength  frequency\n",
+       "0         N      0-1        0.5\n",
+       "1       NNE      0-1        0.6\n",
+       "2        NE      0-1        0.5\n",
+       "3       ENE      0-1        0.4\n",
+       "4         E      0-1        0.4"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.express as px\n",
+    "from plotly import data\n",
+    "\n",
+    "df = data.wind()\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48f725e0",
+   "metadata": {},
+   "source": [
+    "Using the plotting data generated, create a plot and save it to a file. Utilizing `write_image` we can save our plot to a file. We can use this file to later log with  `Rubicon_ml`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "31e007bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_plot=px.scatter(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "scatter_plot.write_image(\"saved-scatter-plot-for-logging.png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a93ac1ec",
+   "metadata": {},
+   "source": [
+    "We can also create a plot and covert it to bytes to save it in memory. Using `to_image` we can convert our plot to bytes in-memory. We'll later log the plot from in-memory with `Rubicon-ml`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8544305b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar_plot=px.bar(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "bar_plot_bytes=bar_plot.to_image(format=\"png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a41ac4b",
+   "metadata": {},
+   "source": [
+    "## Logging your plot\n",
+    "In `Rubicon_ml` , `artifacts` are a catch-all for any other type of data that can be logged to a file. We can use artifacts to capture information that doesn't fall into the `experiment`, `dataframe`, `feature`, and  `parameter` category. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bc3f6aa",
+   "metadata": {},
+   "source": [
+    "Leveraging `artifacts`, plots can be logged just by passing its file location in the `data_path` argument to `project.log_artifact()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d0823f93",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<rubicon_ml.client.artifact.Artifact at 0x16624a1d0>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "project.log_artifact(data_path=\"saved-scatter-plot-for-logging.png\", description=\"bar plot logged with path\")\n",
+    "artifact_plot_from_file=project.artifacts()[0]\n",
```

Comment:
instead of indexing into the list of artifacts, can you give the logged artifact a name and retrieve it with `project.get_artifact(name=name)`?